apps/client/src/plugins/Measurer/Segment.js

```diff
@@ -0,0 +1,289 @@
+import { Style, Text } from "ol/style";
+import { Feature } from "ol";
+import { LineString, Point, Polygon } from "ol/geom";
+
+export default class Segment {
+  #enabled;
+  #drawModel;
+  #currentSegmentLength;
+  #polygonFirstAndLast;
+  #currentCenter;
+  #measureFeature;
+  #map;
+  #currentFeatureSavedId;
+  #currentPoint;
+
+  constructor(drawModel, map, enabled) {
+    this.#enabled = enabled | false;
```

Comment:
Interesting with the bitwise OR here. Should get the job done. However, the nullish coalescing operator is common in Hajk, so let's stick to that to avoid confusion.

```js
this.#enabled = enabled ?? false;
```